src/importer/types.ts

```diff
@@ -57,10 +63,11 @@ export interface ImporterState {
   validationErrors: ImporterValidationError[];
   sheetData: SheetState[];
   parsedFile?: ParsedFile;
-  rowFile?: File;
+  fileData?: FileData;
   columnMappings?: ColumnMapping[];
   importProgress: number;
   importStatistics?: ImportStatistics;
+  indexDBConfig?: IndexDBConfig;
```

Comment:
Why do we need to store it in the state?

Reply:
so we can see in the reducer if we need to save it in the indexDB

Reply:
But we could pass it as props - see my proposal here - https://github.com/HelloCSV/HelloCSV/pull/142#issuecomment-2858932498

Reply:
Plus my proposal also saves state to DB as useEffect instead of doing it in the reducer - I think thats a cleaner solution. Reducer should only generate a new state - it shouldn't have any side effects - especially not async ones

Reply:
added updates to the file